### PR TITLE
Bump lifecycle version from 0.17.3 to 0.19.7

### PR DESCRIPTION
**Pull request by natalieparellano**

### CODE REVIEW

**Review by samj1912 — APPROVED** [review]

**Comment by samj1912** [comment]
From slack - 

> Given this bump removes a bunch of buildpack api versions - can we keep it for another release?

**Comment by natalieparellano** [comment]
@samj1912 fine with me!

**Comment by natalieparellano** [comment]
We now have 0.17.7. Would that be preferred? @tomkennedy513 @chenbh @samj1912

File: hack/lifecycle/main.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ import (
 const (
 	lifecycleMetadataLabel = "io.buildpacks.lifecycle.metadata"
 	lifecycleLocation      = "/cnb/lifecycle/"
-	lifecycleVersion       = "0.17.3"
+	lifecycleVersion       = "0.19.7"
```
Comment (natalieparellano) [inline hack/lifecycle/main.go:35]:
```suggestion
	lifecycleVersion       = "0.20.0"
```

**Comment by k8sVoodoo** [comment]
Sweet thank you for this! I will keep a lookout for the next release that implements this change 😁

**Pull request merged**